pkg/cmd/secret/shared/shared.go

```diff
@@ -1,9 +1,81 @@
 package shared
 
+import "strings"
+
 type Visibility string
 
 const (
 	All      = "all"
 	Private  = "private"
 	Selected = "selected"
 )
+
+type App string
+
+const (
+	Actions    = "actions"
+	Codespaces = "codespaces"
+	Dependabot = "dependabot"
+	Unknown    = "unknown"
+)
+
+func (app App) String() string {
+	return string(app)
+}
+
+func (app App) Title() string {
+	return strings.Title(app.String())
+}
+
+type SecretEntity string
+
+const (
+	Repository   = "repository"
+	Organization = "organization"
+	User         = "user"
+	Environment  = "environment"
+)
+
+func GetSecretEntity(orgName, envName string, userSecrets bool) SecretEntity {
```

Comment:
Since there can only be one secret entity, should this return some sort of error or error state when more than one input is set to a truthy value?

Reply:
I made this error in https://github.com/cli/cli/pull/5134/commits/5d4b163975d88ba82afbeb2cca184612c829adc5.
This is a change in behavior though, but it's for a case that was previously undefined.